packages/atomic/src/components/common/atomic-icon/atomic-icon.tsx

```diff
@@ -61,12 +66,29 @@ export class AtomicIcon implements InitializableComponent<AnyBindings> {
     }
   }
 
+  private isUrlSVG(url: string) {
+    return /.+\.svg$/i.test(url);
+  }
```

Comment:
Instead of validating the URL, I would prefer to validate the response, since somebody could have an URL that returns an SVG without the extension like `https://mywebsite.invalid/user/benjamin/favorite-emoji`.

Reply:
Good point!